sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotCallGCSuppressFinalize.NetCore.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Tests.Diagnostics
+{
+    public class Custom : IAsyncDisposable
```

Comment:
We usually add a link to the original issue with reproducers for reference
```suggestion
	// https://github.com/SonarSource/sonar-dotnet/issues/3639
    public class Custom : IAsyncDisposable
```